apps/omg/test/support/conformance/property.ex

```diff
@@ -0,0 +1,242 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule Support.Conformance.PropertyGenerators do
```

Comment:
It's hard for me to gauge what's understandable here and what is not, having played around with propcheck now. Pls request comments / documentation as you deem necessary to get a general grip of what's going on (without having to revert https://hexdocs.pm/propcheck/PropCheck.html for details)